server/reports/expiring-items/1_0/template.html

```diff
@@ -0,0 +1,102 @@
+<style>{% include "style.css" %}</style>
+
+<div class="container">
+  <table>
+    <thead>
+      <tr class="heading">
+        <td>Code</td>
```

Comment:
Do we want the translated version here?? I'm guessing we do (although can do that later?)